pages/dug_10/4-osfv-status.md

```diff
@@ -0,0 +1,374 @@
+#### <center> Difference </center>
+
+<center>
+<img src="/img/dug_10/osfv-status/test_counts_difference.png" width=600/>
+</center>
+
+Total count increased by **415**
+
+---
+
+# <center> New test cases </center>
+
+* Added FBT suite testing the fast boot feature
+* Added ACPI suite testing ACPI drivers
+* Added USC suite testing the Always On USB feature
+* Added multiple benchmark test suites:
+  + UPP, CPP, ETHPERF, DIO
+  + testing the CPU, Ethernet and disks performance
+* Automated the CBO - Custom Boot Order test suite
+* Automated most of the AUD tests
+* Extended NVM suite with an NVMe on x2 PCIe test
+* Extended TPM suite with tests for the TPM Physical Presence Interface and changing the Endorsement Primary Seed
+* Extended UTC Docking station tests with variants for every docking station, ME enabled/disabled
+
+<!--
+Graph showing differences since the last status presentation.
+List which suites were updated and short description, like
+"new suite" or "4 new cases"
+-->
+
+---
+
+# <center> New platforms </center>
+
+* Protectli VP2440
+<!--
+Graphs probably won't be necessary.
+List new supported platforms if any
+-->
+
+---
+layout: cover
+background: /intro.png
+class: text-center
+---
+
+# Key changes
+
+---
+
+# <center> Fedora support </center>
+
+Most test cases are also possible run on Fedora.
+The test ID naming convention has changed
+[Tests naming convention](https://github.com/Dasharo/open-source-firmware-validation/blob/develop/docs/tests-naming-convention.md)
+
+`<suite_id><case_id>.<environment_id>`
+* `suite_id` - It's typically 3 to 4 letters, uppercase, identifies the test suite. Should be related to what the test suite accomplishes.
+* `case_id` - It's a three digit number with leading zeros. Identifies test cases in a test suite. Typically test cases are numbered incrementally starting from 001
+* `environment_id` - A three digit number, unambiguously identifies the environment in which the test case is performed. The IDs of environments are separated into groups based on the leading digit:
+  * `1xx` - Firmware,  Example: 101 - EDK2 UEFI
+  * `2xx` - Linux,  Example: 201 - Ubuntu
+  * `3xx` - Windows
+
+---
+
+# <center> Fedora support </center>
+
+<center><img src="/img/dug_10/osfv-status/fedora_percent.png" width=700></center>
+
+<!--
+many test cases can also be performed on Fedora
+every test case *Will* be possible to be performed on Fedora
+
+graphs? stats? % per module?
+
+-->
+
+---
+
+# <center> DeGoogle </center>
+
+Dasharo Test & Feature Matrix at Google Docs deprecated
+
+<img src="/img/dug_10/osfv-status/dtfm.png"/>
+
+<!--
+Dasharo Test & Feature matrix going deprecated and is no longer used.
+Test results published at osfv-results
+-->
+
+---
+
+# <center> OSFV results </center>
+
+The test results are now being published on the [OSFV Results repo](https://github.com/Dasharo/osfv-results/)
+
+<center><img src="/img/dug_10/osfv-status/osfv-results-readme.png" width=700/></center>
```

Comment:
...  if you presented a table from spreadsheet before, now you could showcase how the tests results are presented to the user in this new approach.

This graphic may be also useful, but comparing these 2 end-user interfaces (old and new) would be interesting.